projects/habitat_ovmm/configs/agent/floorplanner_eval.yaml

```diff
@@ -41,7 +41,7 @@ AGENT:
 
   SKILLS:
     GAZE_OBJ:
-      type: oracle #gaze #end_to_end #heuristic #hardcoded
+      type: rl #end_to_end #heuristic #hardcoded
```

Comment:
should we add oracle-sim results to the paper?

Reply:
It might not be necessary in the main results table, but could be useful to analyze the performance of each skill in isolation.

Reply:
yeah i think it would be great